www/base/src/app/common/directives/builds/buildstable.tpl.jade

```diff
@@ -15,10 +15,16 @@
           td(ng-show="builders") {{ builders.get(build.builderid).name }}
           td
             a.bb-buildid-link(ui-sref="build({builder:build.builderid, build:build.number})")
-              span.badge-status(ng-class="results2class(build, 'pulse')")
-                  span.badge-inactive
-                    | {{build.number}}
-                  span.badge-active {{results2text(build)}}
+                script(type="text/ng-template" id="buildsummarytooltip")
+                    buildsummary(buildid="build.buildid" type="tooltip")
+                span.badge-status(uib-tooltip-template="'buildsummarytooltip'"
```

Comment:
the double quote sounds weird. why is that?

Reply:
The single quotes inside of the double quotes are necessary for the uib-tooltip-template to display correctly.  I followed the tooltip code sample at https://angular-ui.github.io/bootstrap/